driver-core/src/main/com/mongodb/internal/operation/WriteConcernHelper.java

```diff
@@ -61,21 +59,18 @@ public static boolean hasWriteConcernError(final BsonDocument result) {
     }
 
     public static MongoWriteConcernException createWriteConcernException(final BsonDocument result, final ServerAddress serverAddress) {
-        MongoWriteConcernException writeConcernException = new MongoWriteConcernException(
+        return new MongoWriteConcernException(
                 createWriteConcernError(result.getDocument("writeConcernError")),
-                WriteConcernResult.acknowledged(0, false, null), serverAddress);
-        result.getArray("errorLabels", new BsonArray()).stream().map(i -> i.asString().getValue())
-                .forEach(writeConcernException::addLabel);
-        return writeConcernException;
+                WriteConcernResult.acknowledged(0, false, null),
+                serverAddress,
+                result.getArray("errorLabels", new BsonArray()));
     }
 
     public static WriteConcernError createWriteConcernError(final BsonDocument writeConcernErrorDocument) {
         return new WriteConcernError(writeConcernErrorDocument.getNumber("code").intValue(),
                 writeConcernErrorDocument.getString("codeName", new BsonString("")).getValue(),
                 writeConcernErrorDocument.getString("errmsg").getValue(),
-                writeConcernErrorDocument.getDocument("errInfo", new BsonDocument()),
-                writeConcernErrorDocument.getArray("errorLabels", new BsonArray()).stream().map(i -> i.asString().getValue())
-                        .collect(Collectors.toSet()));
+                writeConcernErrorDocument.getDocument("errInfo", new BsonDocument()));
```

Comment:
What was the rationale for changing this?  The property is deprecated, but until it's removed it should continue to work as it does.